### ISSUE TITLE
Link to a specific tab

### ISSUE THREAD

**Issue by LorenzoSapora** [saintsystems/nova-linkable-metrics#54]
Hey,

Using the tab package, the currently selected tab is reflected into the url ( `/resources/users/123/#Campaign=sent` ).

Is it currently possible to link to that in this version? If so, point it out and I'll write a PR for the README. If not, tag this with a `feature-request`.

Thanks

**Comment by anderly** [comment]
@LorenzoSapora, will take a look at this. We're using the `Link` element from Inertia (which Nova uses internally) but it doesn't automatically include the currently selected tab (or any URL hash for that matter) in all routes. That's probably something that would could allow for as an explicit opt-in vs. default behavior as there are cases where it wouldn't make sense.

**Comment by LorenzoSapora** [comment]
Yeah, I've been digging, too

```php
                ->route('nova.pages.detail', 
                    [
                        'resource' => 'users', 
                        'resourceId' => $request->resourceId,
                        'Campaign' => 'sent'
                    ])
```

is close with `/resources/users/123/?Campaign=sent` but can't seem to get it to use the hashtag.

**Comment by anderly** [comment]
Yeah, the `route` method is just using the Laravel `route` helper method under the covers which won't include URL hashes. So, most likely, we'd either need to wrap the Link component with a custom component and then provide a method to opt-in to hash inclusion off the Linkable metric api, which we'd then consume in the component to retrieve the current hash and append to the generated URL.

The other possibility is simply use the `url()` method instead of the `route` method and then you have full control over the generated URL. However, you'd still have the challenge of knowing the current URL hash on the PHP side.
